.trivyignore

```diff
@@ -11,6 +11,9 @@ CVE-2023-25173
 CVE-2022-3821
 CVE-2022-41723
 
-# ubuntu base vulnerabilities
-CVE-2022-48303
-CVE-2022-3821
+CVE-2022-41723
+CVE-2022-24795
+CVE-2021-20270
```

Comment:
come of the vulnerabilities are high level, are those planned be addressed by mariner team?

Reply:
We will ask the team to upgrade telegraf